simapp/v2/export.go

```diff
@@ -22,8 +27,39 @@ func (app *SimApp[T]) ExportAppStateAndValidators(jailAllowedAddrs []string) (v2
 		return v2.ExportedApp{}, err
 	}
 
+	// get the current bonded validators
+	resp, err := app.Query(ctx, 0, latestHeight, &stakingtypes.QueryValidatorsRequest{
+		Status: stakingtypes.BondStatusBonded,
+	})
+
+	vals, ok := resp.(*stakingtypes.QueryValidatorsResponse)
+	if !ok {
+		return v2.ExportedApp{}, fmt.Errorf("invalid response, expected QueryValidatorsResponse")
+	}
+
```

Comment:
It looks like the input argument to this func `jailAllowedAddrs []string` is not used anywhere, should there be filtering done somewhere in this func?